LandSandBoat/server
# [Mission] CoP 8-2 IxGhrah Mecanics + Ghrah Mixin

**_I affirm:_**

## What does this pull request do?

Adds mechanics for the CoP NM Ix'Ghrah
Cleans Ghrah Mob Luas and converted mechanics to family mixin

## Steps to test these changes
Progress in CoP A Fate Decided and Pop Ix'Ghrah - Verify Mechanics
Observe Ghrahs Both in and out of combat in Huxzoi and Garder of Ruhumet
